charts/hlf-k8s/README.md

```diff
@@ -25,30 +25,34 @@ The following table lists the configurable parameters of the hlf-k8s chart and d
 | `hlf-peer.ingress.tls` | Peer ingress TLS configuration | (undefined) |
 | `hlf-peer.ingress.hosts` | Peer ingress hosts | (undefined) |
 | `hlf-peer.persistence.enabled` | If true, enable persistence for the Peer | `false` |
 | `hlf-peer.persistence.size` | Size of data volume | (undefined) |
 | `hlf-peer.persistence.storageClass` | Storage class of backing PVC | (undefined) |
-| `chaincodes` | The chaincodes to install on the Peer. See [Install a chaincode](#install-a-chaincode). | `[]` |
-| `chaincodes[].name` | The name of the chaincode | (undefined) |
-| `chaincodes[].version` | The chaincode version | (undefined) |
-| `chaincodes[].src` | The URL to a chaincode archive (.tar.gz) | (undefined) |
-| `chaincodes[].hostPath` | A host path containing the chaincode source code | (undefined) |
-| `chaincodes[].configMap.name` | The name of a ConfigMap containing the chaincode source code (tar.gz) | (undefined) |
-| `chaincodes[].configMap.fileName` | The name of the archive within the ConfigMap | (undefined) |
 | `appChannels` | The application channels to create | `[{channelName: mychannel}]` |
 | `appChannels[].channelName` | The name of the application channel. Must be alphanumerical (9 characters max.) | (undefined) |
 | `appChannels[].organizations` | The organizations to add to the application channel. See [Add an organization to the application channel](#add-an-organization-to-the-application-channel). | `[]` |
 | `appChannels[].proposalOrganizations` | The organizations to fetch signed application channel update proposals from. | `[]` |
-| `appChannels[].chaincodePolicy` | The chaincode policy | (undefined) |
-| `appChannels[].chaincodeName` | The chaincode name | (undefined) |
-| `appChannels[].chaincodeVersion` | The chaincode version | (undefined) |
-| `appChannels[].policies` | This value, if set, will override the default HLF application channel policy. See [Add an organization to the application channel](#add-an-organization-to-the-application-channel). | (undefined) |
+| `appChannels[].channelPolicies` | This value overrides the default HLF channel policy. | (defined in values.yaml) |
+| `appChannels[].appPolicies` | This value overrides the default HLF application policy. | (defined in values.yaml) |
+| `appChannels[].chaincodes` | The chaincodes to install on the Peer. See [Install a chaincode](#install-a-chaincode). | `[]` |
+| `appChannels[].chaincodes[].chaincodeName` | The name of the chaincode | (undefined) |
+| `appChannels[].chaincodes[].chaincodeVersion` | The chaincode version | (undefined) |
+| `appChannels[].chaincodes[].chaincodeAddress` | The URL to the chaincode service | (undefined) |
+| `appChannels[].chaincodes[].chaincodePort` | The port to the chaincode service | (undefined) |
+| `appChannels[].chaincodes[].chaincodeAddress` | The URL to a chaincode service | (undefined) |
+| `appChannels[].chaincodes[].chaincodePolicy` | The chaincode policy | (undefined) |
```

Comment:
I would just name them name/version/address/etc. without the "chaincode" prefix as you did for the image.